From 26bc003c330544b2ff61c6ab19aa2f45030d1917 Mon Sep 17 00:00:00 2001
From: Zach Brown <Zbob750@live.com>
Date: Mon, 28 Jul 2014 03:26:50 -0500
Subject: [PATCH] Stop using spigot's website for timings

There's no reason to do so at all, the fact that it just exists as a redirect is both odd and unnecessary

diff --git a/src/main/java/org/bukkit/command/defaults/TimingsCommand.java b/src/main/java/org/bukkit/command/defaults/TimingsCommand.java
index 7907021..94ce5dc 100644
--- a/src/main/java/org/bukkit/command/defaults/TimingsCommand.java
+++ b/src/main/java/org/bukkit/command/defaults/TimingsCommand.java
@@ -94,7 +94,7 @@ public class TimingsCommand extends BukkitCommand {
                 }
 
                 sender.sendMessage("Timings written to " + timings.getPath());
-                sender.sendMessage( "Paste contents of file into form at http://www.spigotmc.org/go/timings to read results." );
+                sender.sendMessage( "Paste contents of file into form at http://aikar.co/timings.php to read results." );
 
             } catch (IOException e) {
             } finally {
@@ -238,7 +238,7 @@ public class TimingsCommand extends BukkitCommand {
 
                 String location = con.getHeaderField( "Location" );
                 String pasteID = location.substring( "http://paste.ubuntu.com/".length(), location.length() - 1 );
-                sender.sendMessage( ChatColor.GREEN + "View timings results can be viewed at http://www.spigotmc.org/go/timings?url=" + pasteID );
+                sender.sendMessage( ChatColor.GREEN + "View timings results can be viewed at http://aikar.co/timings.php?url=" + pasteID );
             } catch ( IOException ex )
             {
                 sender.sendMessage( ChatColor.RED + "Error pasting timings, check your console for more information" );
-- 
1.9.1

